### PR TITLE
fix: v8 ComboBox preventDismissOnEvent does not override passed-in prop

### PR DESCRIPTION
**Pull request by smhigley**
A fix for a team requires passing in `preventDismissOnEvent` to `calloutProps` in ComboBox, but it is currently overridden by the internal `preventDismissOnEvent`. Quick fix to respect the passed-in prop.

### CODE REVIEW

File: change/@fluentui-react-1c4645e3-a3e3-49b1-9e15-df952be33941.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-1c4645e3-a3e3-49b1-9e15-df952be33941.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29737&commitId=1de79bf8dafc0c2d3b93bf7d96eaf372c4fdf7f6&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/ceca5c9b403eb339e7d359b80329f8691aae2898'>ceca5c9b403eb339e7d359b80329f8691aae2898</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/BaseButton-mount_base.html">643</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/BaseButton-mount.html">630</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Breadcrumb-mount_base.html">1679</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Breadcrumb-mount.html">1679</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Checkbox-mount_base.html">1699</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Checkbox-mount.html">1691</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/CheckboxBase-mount_base.html">1472</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/CheckboxBase-mount.html">1478</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/ChoiceGroup-mount_base.html">2908</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/ChoiceGroup-mount.html">2930</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/ComboBox-mount_base.html">649</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/ComboBox-mount.html">666</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/CommandBar-mount_base.html">6211</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/CommandBar-mount.html">6175</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/ContextualMenu-mount_base.html">13521</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/ContextualMenu-mount.html">13584</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/DefaultButton-mount_base.html">773</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/DefaultButton-mount.html">767</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/DetailsRow-mount_base.html">2185</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/DetailsRow-mount.html">2202</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/DetailsRowFast-mount_base.html">2190</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/DetailsRowFast-mount.html">2174</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/DetailsRowNoStyles-mount_base.html">2043</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/DetailsRowNoStyles-mount.html">2014</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Dialog-mount_base.html">2667</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Dialog-mount.html">2596</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/DocumentCardTitle-mount_base.html">243</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/DocumentCardTitle-mount.html">234</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Dropdown-mount_base.html">1983</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Dropdown-mount.html">1928</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/FocusTrapZone-mount_base.html">1123</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/FocusTrapZone-mount.html">1131</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/FocusZone-mount_base.html">1096</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/FocusZone-mount.html">1056</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/GroupedList-mount_base.html">41617</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/GroupedList-mount.html">41706</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/GroupedList-virtual-rerender_base.html">19832</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/GroupedList-virtual-rerender.html">20064</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">50717</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/GroupedList-virtual-rerender-with-unmount.html">50890</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/GroupedListV2-mount_base.html">231</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/GroupedListV2-mount.html">233</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/GroupedListV2-virtual-rerender_base.html">217</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/GroupedListV2-virtual-rerender.html">208</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">237</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">235</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/IconButton-mount_base.html">1111</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/IconButton-mount.html">1064</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Label-mount_base.html">341</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Label-mount.html">351</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Layer-mount_base.html">2727</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Layer-mount.html">2662</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Link-mount_base.html">393</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Link-mount.html">389</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/MenuButton-mount_base.html">950</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/MenuButton-mount.html">934</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/MessageBar-mount_base.html">21318</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/MessageBar-mount.html">21433</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Nav-mount_base.html">1948</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Nav-mount.html">1898</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/OverflowSet-mount_base.html">767</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/OverflowSet-mount.html">776</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Panel-mount_base.html">1793</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Panel-mount.html">1773</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Persona-mount_base.html">736</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Persona-mount.html">730</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Pivot-mount_base.html">889</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Pivot-mount.html">879</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/PrimaryButton-mount_base.html">854</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/PrimaryButton-mount.html">860</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Rating-mount_base.html">4517</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Rating-mount.html">4582</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/SearchBox-mount_base.html">878</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/SearchBox-mount.html">892</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Shimmer-mount_base.html">1843</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Shimmer-mount.html">1889</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Slider-mount_base.html">1331</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Slider-mount.html">1295</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/SpinButton-mount_base.html">2824</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/SpinButton-mount.html">2835</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Spinner-mount_base.html">368</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Spinner-mount.html">388</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/SplitButton-mount_base.html">1804</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/SplitButton-mount.html">1816</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Stack-mount_base.html">413</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Stack-mount.html">410</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/StackWithIntrinsicChildren-mount_base.html">841</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/StackWithIntrinsicChildren-mount.html">845</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/StackWithTextChildren-mount_base.html">2575</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/StackWithTextChildren-mount.html">2572</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/SwatchColorPicker-mount_base.html">6099</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/SwatchColorPicker-mount.html">6129</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/TagPicker-mount_base.html">1452</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/TagPicker-mount.html">1450</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Text-mount_base.html">368</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Text-mount.html">374</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/TextField-mount_base.html">916</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/TextField-mount.html">907</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/ThemeProvider-mount_base.html">832</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/ThemeProvider-mount.html">822</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/ThemeProvider-virtual-rerender_base.html">587</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/ThemeProvider-virtual-rerender.html">587</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1262</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1245</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Toggle-mount_base.html">619</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/Toggle-mount.html">610</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/buttonNative-mount_base.html">188</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29737/perf-test/buttonNative-mount.html">193</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Review by khmakoto — APPROVED** [review]

**Pull request merged**